### ISSUE TITLE
Cannot use class `.selectors` when using with postcss-modules

### ISSUE THREAD

**Issue by JakeStanger** [postcss/autoprefixer#1269]
Using a class name of `.selectors` when running with `postcss-modules` then `autoprefixer` causes the following error:

```
UnhandledPromiseRejectionWarning: TypeError: checker.check is not a function
```

This happens at [this line](https://github.com/postcss/autoprefixer/blob/master/lib/processor.js#L468).

Minimal reproduction:

```js
const autoprefixer = require("autoprefixer");
const cssModules = require("postcss-modules");
const postcss = require("postcss");

const css = `
.selectors {
  color: red;
}
`;

postcss([
  cssModules,
  autoprefixer
])
  .process(css)
  .then(result => console.log(result.css));
```

Sticking a log above the erroring line to show `checker` yields the following:

```
OldValue {
  unprefixed: 'calc',
  prefixed: '-webkit-calc',
  string: '-webkit-calc',
  regexp: /(^|[\s,(])(\-webkit\-calc($|[\s(,]))/gi }
OldValue {
  unprefixed: 'calc',
  prefixed: '-moz-calc',
  string: '-moz-calc',
  regexp: /(^|[\s,(])(\-moz\-calc($|[\s(,]))/gi }
OldValue {
  unprefixed: 'calc',
  prefixed: '-webkit-calc',
  string: '-webkit-calc',
  regexp: /(^|[\s,(])(\-webkit\-calc($|[\s(,]))/gi }
OldValue {
  unprefixed: 'calc',
  prefixed: '-moz-calc',
  string: '-moz-calc',
  regexp: /(^|[\s,(])(\-moz\-calc($|[\s(,]))/gi }
[Function: values] // Looks like the checker is being overriden by a function
```

**Comment by ai** [comment]
Fixed 33ee989

**Comment by ai** [comment]
Released in 9.7.3

**Comment by JakeStanger** [comment]
Awesome thanks for getting this fixed so fast
